src/core/state.go

```diff
@@ -669,6 +674,37 @@ func (state *BuildState) WaitForPackage(label BuildLabel) *Package {
 	return state.Graph.PackageByLabel(label) // Important to check again; it's possible to race against this whole lot.
 }
 
+// WaitForPackage is similar to WaitForBuiltTarget however
```

Comment:
what a cliffhanger!

Reply:
You'll just have to wait for the rest of this message